cs-config/install.sh

```diff
@@ -0,0 +1 @@
+conda update env --name base -f environment.yml
```

Comment:
@MaxGhenis The newest version of `cs-kit` has a command `build-env` for detecting the `environment.yml` file and building the environment accordingly:

```suggestion
csk build-env
```

Once this is tested a little more, I'll make running this command the default.

Note: `cs-kit` is installed already. So there's no need to install it before running this command.